README.md

```diff
@@ -175,3 +176,61 @@ See also [reference documentation](https://taskcluster.github.io/fast-azure-stor
  * `Queue#updateMessage(queue, text, messageId, popReceipt, options)`
  * `Queue#sas(queue, options)`
 
+Azure Blob Storage Client
+--------------------------
```

Comment:
In Markdown there are 2 ways to specify H2 headers.
1) using ## in front of text 
2) using underline style (my case)
I also wanted to be consistent with the other headers from the README document